.github/workflows/ui-tests/cypress/integration/create_spec.js

```diff
@@ -28,6 +28,53 @@ describe('User can navigate to create account screen', function() {
     })
 })
 
+describe('Signup is unsuccessful with long username', function() {
+    const invalidFeedback = Cypress.env('messages').invalidFeedback
+
+    beforeEach(function() {
+        const user = {
+            username: '0123456789abcdefghijklmnopqrstuvwxyz',
+            firstName: 'Tom',
+            lastName: 'Nook',
+            password: 'bells'
+        }
+        cy.createAccount(user)
+    })
+
+    it('stay on signup page', function() {
+        cy.url().should('include', '/signup')
+    })
+
+    it('error message visible', function() {
+        cy.get('.invalid-feedback').should('be.visible')
+        cy.get('.invalid-feedback').contains(invalidFeedback.username)
+    })
+})
+
+describe('Signup is unsuccessful with non-alphanumeric username', function() {
+    const invalidFeedback = Cypress.env('messages').invalidFeedback
+
+    beforeEach(function() {
+        const user = {
+            username: 'Tom@Nook',
```

Comment:
same as above, making this random would make it consistent

Reply:
What kind of randomness are you thinking here? I'd be cautious about trying random non-alphanumeric characters, because we want the tests to be consistent.

This would be another good use of a loop if we wanted to test a bunch of bad strings. But that should be covered in the unit tests at least

Reply:
no worries,  since it's covered in unit test already leaving as is seems reasonable